sdk-setup/src/sdk-manage

```diff
@@ -611,6 +611,21 @@ object_upgradable() {
     object_zypper "$type:$object" --quiet --no-refresh list-updates
 }
 
+rebuild_rpmdb() {
+    local object=$1
+    # rpm can't overwrite directory on a different overlay layer
+    if mount | grep -q "overlay on / type overlay"; then
+        enter_target "$object" rpm --rebuilddb > /dev/null 2&>1
+        local new_db=$(enter_target "$object" ls /var/lib|grep rpmrebuild)
+        if [[ ! -z $new_db ]]; then
+            enter_target "$object" rm -Rf /var/lib/rpm
+            enter_target "$object" mv /var/lib/$new_db /var/lib/rpm
+        fi
+    else
+        enter_target "$object" rpm --rebuilddb
+    fi
```

Comment:
```suggestion
    if [[ $(findmnt --noheadings --output FSTYPE / ) == overlay ]]; then
        enter "$object" rpm --rebuilddb &>/dev/null
        local new_db=$(enter "$object" find /var/lib -maxdepth 1 -name 'rpmrebuilddb.*' |head -n1)
        if [[ $new_db ]]; then
            enter "$object" rm -rf /var/lib/rpm || return
            enter "$object" mv "$new_db" /var/lib/rpm
        fi
    else
        enter "$object" rpm --rebuilddb
    fi
```

Reply:
Why the change to use enter instead of enter_target?

Reply:
Given its name, position in the file and (most importantly) effect, it is a generic function. No matter that we only use it with targets right now.